javascript/modules/client/src/internal/interfaces/client.ts

```diff
@@ -160,6 +160,8 @@ export type DaoCreationStepValue =
 // DEPOSIT
 
 export interface IDepositParams {
+  // TODO
+  // update naming to daoAddressOrEns
   daoAddress: string;
```

Comment:
Why don't we change it now, then? The UI is only going to get more coupled than it is now